062-support-tiered-storage.md

```diff
@@ -0,0 +1,145 @@
+* Currently, tiered storage is not available for compacted topics.
+* Once Tiered Storage is enabled for a topic, it becomes a permanent configuration and cannot be disabled.
+* Disabling Tiered Storage for the entire cluster requires manual deletion of all topics using Tiered Storage.
+* All Kafka clients, regardless of their version, can continue to produce and consume records from topics utilizing Tiered Storage. However, clients with versions earlier than 3.0 are limited in performing administrative actions, such as enabling Tiered Storage on a topic
+
+All the limitations above except the client compatibility are expected to be fixed before it is production-ready in Kafka.
+
+We choose not to validate the error conditions at the Strimzi operator layer because:
+
+* At least some of them are expected to be removed in the future
+* Validating them might be quite complex (for example, IIRC, the compacted topics can be set in the topic itself as well as in the broker defaults)
+* Validating this in UTO would capture only a part of the topics and not prevent the users from misconfiguration. So they will still need to be aware of these rules and learn to respect them.
+
+### Docker image
+
+This proposal doesn't include built in plugin support in any Strimzi built Kafka image. Users need to specify the path via the config, and customize the docker image to include the library.
+
+An example to build such custom image is via a custom Dockerfile:
+
+```
+FROM quay.io/strimzi/kafka:0.40.0-kafka-3.6.0
+ARG REMOTE_PLUGIN_PATH
+COPY ${REMOTE_PLUGIN_PATH} /opt/kafka/plugins/tiered-storage-s3/
+```
+
+An image with plugin library can be built with the following command:
+
+```
+docker build -t docker.example.com/tierstorage/kafka:0.40.0-kafka-3.6.0-plugin-1 . -build-arg REMOTE_PLUGIN_PATH=./s3/build/install/s3/*
+```
+## Testing strategy
+
+Unit tests will be added to validate that all the RLM, RSM, and RLMM changes are propagated through.
+Because the RSM and RLMM are typically provided by user and there is no built in support for any third party cloud provider, there won’t be any integration tests covering the cloud provider interaction. It’s the user’s responsibility to ensure the RSM and RLMM works with the intended cloud provider.
+
+For testing purposes, a system test with `LocalTieredStorage` implementation can be added to ensure end-to-end functionality.
+
+## Affected Projects
+
+Strimzi operator will be affected with the change.
+
+## Compatibility
+
+This proposal adds a new API and configuration, so there are no backward compatibility issues with the existing APIs.
+
+## Alternate consideration
+
+A simple way is to set everything via Helm chart config: https://github.com/strimzi/proposals/pull/98
+
+That option is not favored because it's not applicable to usage without helm chart. It might also not be able to guarantee compatibility with future Stirmzi versions.
```

Comment:
Just two typos

```suggestion
That option is not favored because it's not applicable to usage without Helm chart. It might also not be able to guarantee compatibility with future Strimzi versions.
```

Reply:
done